apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -41,14 +52,22 @@ import React, { FunctionComponent, ReactElement, useEffect, useState } from "rea
 import { useTranslation } from "react-i18next";
 import { useDispatch } from "react-redux";
 import { Dispatch } from "redux";
-import { Dropdown, DropdownItemProps, DropdownProps, Form, Grid, Icon, Modal } from "semantic-ui-react";
+import { Button, Dropdown, DropdownItemProps, DropdownProps, Form, Grid, Icon, Modal } from "semantic-ui-react";
```

Comment:
Can we use the `Button` from oxygen.

Reply:
We are using a button group here which is not in oxygen-ui yet.

Reply:
Ack